### ISSUE TITLE
Maven plugin failed to run on basic options (printing run detectors and analyzed classes)

### ISSUE THREAD

**Issue by gloNelson** [spotbugs/spotbugs-maven-plugin#306]
I found a bug when using the maven plugin. For basic options supported by spotbugs (printing run detectors and analyzed classes), it fails to print anything to the log.
### Expected Output
print run detectors and analyzed classes

### Actual Ouput
As can be seen from the output, the analyzed classes are not printed in the log file.
> [java] 2 analysis passes to perform
[java] Pass 1: Analyzing classes (0 / 171) - 00% completePass 1: Analyzing classes (1 / 171) - 00% completePass 1: Analyzing classes (2 / 171) - 01% completePass 1: Analyzing classes (3 / 171) - 01% completePass 1: Analyzing classes (4 / 171) - 02% completePass 1: Analyzing classes (5 / 171) - 02% completePass 1: Analyzing classes (6 / 171) - 03% completePass 1: Analyzing classes (7 / 171) - 04% completePass 1: Analyzing classes (8 / 171) - 04% completePass 1: Analyzing classes (9 / 171) - 05% completePass 1: Analyzing classes (10 / 171) - 05% completePass 1: Analyzing classes (11 / 171) - 06% completePass 1: Analyzing classes (12 / 171) - 07% completePass 1: Analyzing classes (13 / 171) - 07% completePass 1: Analyzing classes (14 / 171) - 08% completePass 1: Analyzing classes (15 / 171) - 08% completePass 1: Analyzing classes (16 / 171) - 09% completePass 1: Analyzing classes (17 / 171) - 09% completePass 1: Analyzing classes (18 / 171) - 10% completePass 1: Analyzing classes (19 / 171) - 11% completePass 1: Analyzing classes (20 / 171) - 11% completePass 1: Analyzing classes (21 / 171) - 12% completePass 1: Analyzing classes (22 / 171) - 12% completePass 1: Analyzing classes (23 / 171) - 13% completePass 1: Analyzing classes (24 / 171) - 14% completePass 1: Analyzing classes (25 / 171) - 14% completePass 1: Analyzing classes (26 / 171) - 15% completePass 1: Analyzing classes (27 / 171) - 15% completePass 1: Analyzing classes (28 / 171) - 16% completePass 1: Analyzing classes (29 / 171) - 16% completePass 1: Analyzing classes (30 / 171) - 17% completePass 1: Analyzing classes (31 / 171) - 18% completePass 1: Analyzing classes (32 / 171) - 18% completePass 1: Analyzing classes (33 / 171) - 19% completePass 1: Analyzing classes (34 / 171) - 19% completePass 1: Analyzing classes (35 / 171) - 20% completePass 1: Analyzing classes (36 / 171) - 21% completePass 1: Analyzing classes (37 / 171) - 21% completePass 1: Analyzing classes (38 / 171) - 22% completePass 1: Analyzing classes (39 / 171) - 22% completePass 1: Analyzing classes (40 / 171) - 23% completePass 1: Analyzing classes (41 / 171) - 23% completePass 1: Analyzing classes (42 / 171) - 24% completePass 1: Analyzing classes (43 / 171) - 25% completePass 1: Analyzing classes (44 / 171) - 25% completePass 1: Analyzing classes (45 / 171) - 26% completePass 1: Analyzing classes (46 / 171) - 26% completePass 1: Analyzing classes (47 / 171) - 27% completePass 1: Analyzing classes (48 / 171) - 28% completePass 1 ...

### My `pom.xml`
```pom.xml
<plugin>
    <groupId>com.github.spotbugs</groupId>
        <artifactId>spotbugs-maven-plugin</artifactId>
        <version>4.1.4</version>
        <configuration>
            <effort>Default</effort>
            <threshold>Low</threshold>
            <debug>true</debug>
            <trace>true</trace>
            <spotbugs.trace>true</spotbugs.trace>
        </configuration>
    <dependencies>
        <dependency>
            <groupId>com.github.spotbugs</groupId>
            <artifactId>spotbugs</artifactId>
            <version>4.1.4</version>
        </dependency>
    </dependencies>
</plugin>

The attachment is my log with -X option.
[execute.log](https://github.com/spotbugs/spotbugs-maven-plugin/files/6154659/execute.log)

**Comment by gloNelson** [comment]
I realized that the fault is in the way the maven plugin sets the property. I have created a pull request to fix this. Please help me review this. @hazendaz Thanks a lot.

**Comment by hazendaz** [comment]
merged in fix #307, thanks for finding the cause of issue.
